README.md

```diff
@@ -8,7 +8,7 @@ It's annoying to get a tarball to install a program. You won't get menu items, p
 
 ## Notes
 
-I've tested this only on Linux Mint 19, but most likely it will work on other Debian based distros too (for example Ubuntu).
+Initially from [@niklas-e](https://github.com/niklas-e). Updated and enhanced with workflow by [@kitingChris](https://github.com/kitingChris)´.
```

Comment:
Could be left out or changed to something like this:

```
## Contributors

- [@niklas-e](https://github.com/niklas-e)
- [@kitingChris](https://github.com/kitingChris)
```

Reply:
sure :)